### PR TITLE
feat: 🎸 add Airtable call to action  button

### PR DESCRIPTION
**Pull request by MatthysDev**
✅ Closes: #139

### Checklist before merge

- [ ] The first commit is nice to make ChatGPT happy
- [ ] Merge the pull request using **Squash and merge**

### CODE REVIEW

File: apps/weshipit/components/AirtableToggleForm.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  return (
+    <>
+      {isFormOpen ? (
+        <iframe
+          src={formLink}
+          width="100%"
+          height={formHeight}
+          className="m-auto rounded-md"
+          style={{ background: '#ffffff' }}
```
Comment (flexbox) [inline apps/weshipit/components/AirtableToggleForm.tsx:29]: can you use tailwind className for `bg-white` and `dark:bg-gray-900` (these are example values replace with the good ones) ?

File: apps/weshipit/components/AirtableToggleForm.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        />
+      ) : (
+        <Button onClick={toggleNavigation} size="xl" className="mb-16 mt-4">
+          <span>{buttonText}</span> <span className="ml-2">→</span>
```
Comment (flexbox) [inline apps/weshipit/components/AirtableToggleForm.tsx:33]: If you want to add an icon use Heroicon

File: apps/weshipit/pages/onboarding.tsx [hunk 3]
```diff
@@ -11,13 +14,65 @@ export default function Onboarding() {
...
+      <Text
+        as="h3"
+        variant="p2"
+        className="my-12 text-center italic opacity-75"
+      >
+        Join tens of innovators that have already worked with me
+      </Text>
+      <ClientsListHomepage clients={clients} />
+      <div className="m-auto mt-24 flex w-1/2 flex-col items-center">
+        <Text as="h3" variant="p2" className=" my-2">
```
Comment (flexbox) [inline apps/weshipit/pages/onboarding.tsx:42]: use Notion ai to create a new text
use `we` instead of `I`

**Review by flexbox — CHANGES_REQUESTED** [review]

**Comment by flexbox** [comment]
@MatthysDev links on the Header should go to this `/onboarding` page

image

**Comment by MatthysDev** [comment]
@flexbox i think we can close it because onboarding page was update since this pr

**Pull request closed without merging**